roles/dokuwiki/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Create DokuWiki Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ dokuwiki_data_directory }}/data"
+
+- name: DokuWiki Docker Container
+  docker_container:
+    name: dokuwiki
+    image: linuxserver/dokuwiki:latest
```

Comment:
Most linuxserver containers accept uid/gid, do we need this here to permission the data correctly?

Reply:
I set the uid/gid to 1000:1000. It works. I wasn't sure about those, so I checked https://hub.docker.com/r/linuxserver/dokuwiki. I understand those Docker permissions better now.